web_widget_text_markdown/__manifest__.py

```diff
@@ -3,29 +3,22 @@
 #
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 {
-    'name': 'Web Widget Text Markdown',
-    'version': '11.0.1.0.0',
+    "name": "Web Widget Text Markdown",
+    "version": "13.0.1.0.0",
```

Comment:
Minor nitpick, but the OCA way is to _first_ do the black/isort/prettier in a separate commit, and then the actual migration to 13.0 in a separate commit, which makes it easier to review (people will just have to look at the last commit). Could you still separate the two?

Reply:
This one I combined since when I made the first commit I had run pre-commit with my module added. But fixed it in [project_task_markdown](https://github.com/OCA/project/pull/828) and also learnt how to add a branch for travils cl in [doc](https://github.com/OCA/project/pull/828)